presto-main/src/main/java/com/facebook/presto/operator/aggregation/GenericAccumulatorFactory.java

```diff
@@ -477,22 +486,27 @@ public void addInput(WindowIndex index, List<Integer> channels, int startPositio
         @Override
         public void addIntermediate(Block block)
         {
-            throw new UnsupportedOperationException();
+            accumulator.addIntermediate(block);
         }
 
         @Override
         public void evaluateIntermediate(BlockBuilder blockBuilder)
         {
-            throw new UnsupportedOperationException();
+            accumulator.evaluateIntermediate(blockBuilder);
```

Comment:
ignore these changes, they are not relevant  to the grouped accumulator which is what I've been focusing on.